### PR TITLE
Add bookings feature

### PR DESCRIPTION
**Pull request by davmcgregor**
Some substantial changes here but all working!

### CODE REVIEW

File: App/views/spaces/details.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+</h2>
+<div>
+    <p><%= @space.name %></p> 
+    <p><%= @space.description %></p> 
+    <p>£<%= @space.price %> per night</p>
+    <form action='/bookings/new' method='post'>
+      <input type='date' name='date_start'>
+      <input type='date' name='date_end'>
+      <input type='submit' value='Book'>
+    </form>
```
Comment (elebecca) [inline App/views/spaces/details.erb:12]: To-do tomorrow: add call to action: make a booking

File: App/views/spaces/index.erb [hunk 2]
```diff
@@ -18,8 +21,14 @@
     <p><%= space.name %></p> 
     <p><%= space.description %></p> 
     <p>£<%= space.price %> per night</p>
+<<<<<<< HEAD
```
Comment (elebecca) [inline App/views/spaces/index.erb]: delete this line

File: App/views/spaces/index.erb [hunk 3]
```diff
@@ -18,8 +21,14 @@
...
+    <form action="/spaces/<%= space.id %>">
+      <input type='submit' value='More Details'>
+    </form>
+=======
```
Comment (elebecca) [inline App/views/spaces/index.erb]: delete this line

File: App/views/spaces/index.erb [hunk 4]
```diff
@@ -18,8 +21,14 @@
...
     <p>Available from <%= space.date_start %></p>
     <p>Available to <%= space.date_end %></p>
+>>>>>>> master
```
Comment (elebecca) [inline App/views/spaces/index.erb]: delete this line :)

**Review by elebecca — CHANGES_REQUESTED** [review]
Everything else is looking good !

File: App/views/spaces/index.erb (hunk 4 above)
Comment (davmcgregor) [reply App/views/spaces/index.erb]: done

File: App/views/spaces/details.erb (hunk 1 above)
Comment (davmcgregor) [reply App/views/spaces/details.erb:12]: ok

**Review by elebecca — APPROVED** [review]

**Pull request merged**